src/lib/Microsoft.Health.Fhir.Ingest/Service/MeasurementImportServiceExtensions.cs

```diff
@@ -1,30 +0,0 @@
-// -------------------------------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
-// -------------------------------------------------------------------------------------------------
-
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.DependencyInjection.Extensions;
-using Microsoft.Health.Fhir.Ingest.Config;
-
-namespace Microsoft.Health.Fhir.Ingest.Service
-{
-    public class MeasurementImportServiceExtensions
-    {
-        public static void AddImportService(IServiceCollection services, IConfiguration configuration)
-        {
-            var collectorOptions = new CollectorOptions();
-            configuration.GetSection(CollectorOptions.Settings).Bind(collectorOptions);
-
-            if (!collectorOptions.UseCompressionOnSend)
-            {
-                services.TryAddSingleton<IImportService, MeasurementFhirImportService>();
```

Comment:
Is there any reason to keep the MeasurementFhirImportService?  It looks like it is used in the legacy Azure Functions but since the new service is backwards compatible it seems to me like we should be able swap them there too.  Thoughts?  If so it wouldn't be for this PR but want to see if we avoid maintaining both services long term.

Reply:
Yeah we should deprecate MeasurementFhirImportService when we deprecate the legacy Azure Functions code, or just swap it out for the new class. Created user story 101315 under the Azure Functions / Stream Analytics deprecation feature 99669.